cmd/config.go

```diff
@@ -0,0 +1,48 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/bianjieai/tibc-relayer-go/cmd/handlers"
+	"github.com/spf13/cobra"
+)
+
+const defaultConfigDir = ".relayer"
```

Comment:
How about using the following defultDir?
```suggestion
const defaultConfigDir = filepath.Join(os.UserHomeDir(), ".tibc-relayer")
```